File: lp2/chatbot.py
def simple_chatbot():
    print("Welcome to Simple Chatbot! (type 'exit' to quit)")

    while True:
        user_input = input("You: ").lower()

        if user_input == "hello" or user_input == "hi":
            print("Bot: Hello! How can I help you?")
        elif user_input == "how are you":
            print("Bot: I'm doing well. Thank you!")
        elif user_input == "what is your name":
            print("Bot: I'm a basic chatbot.")
        elif user_input == "bye":
            print("Bot: Goodbye! Take care.")
            break
        elif user_input == "help":
            print("Bot: You can try saying 'hello', 'how are you', 'what is your name', or 'bye'.")
        else:
            print("Bot: Sorry, I don't understand that.")


simple_chatbot()

File: lp2/chatbot.java
import java.util.*;

class chatbot {
    public static void main(String args[]) {
        System.out.println("------Welcome To Chatbot System-------");
        Scanner sc = new Scanner(System.in);

        boolean ans = true;
        while (ans) {
            System.out.print("You : ");
            String response = sc.nextLine();

            // Convert input to lower case to simplify comparison
            response = response.toLowerCase();

            if (response.equals("hii") || response.equals("hello")) {
                System.out.println("Bot : Hello Buddy! How can I help you?");
            } else if (response.equals("exit")) {
                ans = false;
                System.out.println("Bot : Goodbye!");
            } else {
                System.out.println("Bot : Sorry, I didn't understand that.");
            }
        }

        sc.close();
    }
}
